### PR TITLE
replace underscore with standalone lodash modules

### PR DESCRIPTION
**Pull request by dhritzkiv**
As per http://blog.andyet.com/2015/03/25/amped-down

### CODE REVIEW

File: ampersand-state.js [hunk 1]
```diff
@@ -1,5 +1,29 @@
...
+var keys = require("lodash.keys");
+var bind = require("lodash.bind");
+var defaults = require("lodash.defaults");
+var union = require("lodash.union");
+var BBEvents = require('backbone-events-standalone');
+var KeyTree = require('key-tree-store');
+var arrayNext = require('array-next');
 var BBEvents = require('backbone-events-standalone');
 var KeyTree = require('key-tree-store');
 var arrayNext = require('array-next');
```
Comment (latentflip) [inline ampersand-state.js]: We seem to have required BBEvents/KeyTree/arrayNext twice?

File: ampersand-state.js [hunk 2]
```diff
@@ -537,17 +561,17 @@ function createPropertyDefinition(object, name, desc, isSession) {
             this.set(name, val);
         },
         get: function () {
-            var result = this._values[name];
+            var value = this._values[name];
```
Comment (latentflip) [inline ampersand-state.js]: Was there a reason to change this?

**Comment by latentflip** [comment]
Thanks! Added a couple of comments.

File: ampersand-state.js (hunk 2 above)
Comment (pgilad) [inline ampersand-state.js]: I'm guessing because `result` is used by the `lodash` module...

Comment (dhritzkiv) [inline ampersand-state.js]: ^ yes, this is why.

Comment (latentflip) [inline ampersand-state.js]: Okay, yes I guess that helps disambiguate at least. Thanks

**Comment by wraithgar** [comment]
+1

**Comment by latentflip** [comment]
+1

**Comment by latentflip** [comment]
Thanks @dhritzkiv !

**Comment by dhritzkiv** [comment]
No probs. I'll try to get a few more @AmpersandJS modules converted over the next few days.

**Comment by wraithgar** [comment]
@latentflip you gonna patch release this?

**Comment by wraithgar** [comment]
released as `4.4.6`

**Comment by latentflip** [comment]
Thanks @wraithgar :tada:

**Pull request merged**